test/integ/cli/test_cli_manifest_upload.py

```diff
@@ -111,28 +109,31 @@ def test_manifest_upload_by_farm_queue(self, temp_dir):
         manifest_file = self.create_manifest_file(temp_dir)
         manifest_file_name = Path(manifest_file).name
 
+        # Input:
+        farm_id = os.environ.get("FARM_ID", "")
+        queue_id = os.environ.get("QUEUE_ID", "")
+
         # Now that we have a manifest file, execute the CLI and upload it to S3
         # The manifest file name is unique, so it will not collide with prior test runs.
         s3_bucket = os.environ.get("JOB_ATTACHMENTS_BUCKET", "")
         runner = CliRunner()
-        # Temporary, always add cli_manifest until launched.
-        main.add_command(cli_manifest)
         result = runner.invoke(
             main,
             [
                 "manifest",
                 "upload",
                 "--farm-id",
-                os.environ.get("FARM_ID", ""),
+                farm_id,
                 "--queue-id",
-                os.environ.get("QUEUE_ID", ""),
+                queue_id,
                 manifest_file,
             ],
         )
         assert result.exit_code == 0, f"Non-Zeo exit code, CLI output {result.output}"
 
         # Then validate the Manifest file is uploaded to S3 by checking the file actually exists.
-        manifest_s3_path = f"DeadlineCloud/Manifests/{manifest_file_name}"
+        root_prefix = get_queue(farm_id=farm_id, queue_id=queue_id).jobAttachmentSettings.rootPrefix  # type: ignore[union-attr]
```

Comment:
Here I don't mind using `get_queue`, because its test code and not customer facing. Its good here to keep the test code simpler.